### PR TITLE
Adds desktop and landscape mode

### PR DESCRIPTION
**Pull request by kul3r4**
Fixez #1811

### CODE REVIEW

File: src/stories/20_Features/Desktop_and_landscape.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      </amp-story-page>
+      <amp-story-page id="page-2">
+        <amp-story-grid-layer template="fill">
+          <amp-img src="https://unsplash.it/720/640/?image=11"
+          width="720"
+          height="640" layout="responsive"></amp-img>
+        </amp-story-grid-layer>
+      </amp-story-page>
+
+      <!-- A "bookend" panel containing links to other resources will appear on the last page of your story if you include an `amp-story-bookend` that references a [bookend JSON config](/json/bookend.json). -->
```
Comment (sebastianbenz) [inline src/stories/20_Features/Desktop_and_landscape.html]: I don't think this is needed here.

File: src/stories/20_Features/Desktop_and_landscape.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    - change the desktop experience to an immersive full bleed mode, replacing the default three portrait panels experience -->
+    <amp-story standalone supports-landscape
+      title="Stories in AMP - Hello World"
+      publisher="AMP Project"
+      publisher-logo-src="<%host%>/favicons/coast-228x228.png"
+      poster-portrait-src="<%host%>/img/story_dog2_portrait.jpg"
+      poster-square-src="<%host%>/img/story_dog2_square.jpg"
+      poster-landscape-src="<%host%>/img/story_dog2_landscape.jpg">
+
+      <!-- A story consists of one or more pages. Each page is declared by an `amp-story-page` element. Pages are designed by layering videos, images and text. Here we have a page that uses two layers. One layer filling the available space with an image and one text layer that shows a heading. -->
```
Comment (sebastianbenz) [inline src/stories/20_Features/Desktop_and_landscape.html]: I don't think this comment makes sense for this sample, which is about desktop mode and not a general intro.

File: src/stories/20_Features/Desktop_and_landscape.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      <amp-story-page id="page-1">
+        <amp-story-grid-layer template="fill">
+          <amp-img src="https://unsplash.it/720/640/?image=10"
```
Comment (sebastianbenz) [inline src/stories/20_Features/Desktop_and_landscape.html]: The resolution seems very small for a large desktop.

File: src/stories/20_Features/Desktop_and_landscape.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<!---
+
+skipValidation: 'true'
```
Comment (sebastianbenz) [inline src/stories/20_Features/Desktop_and_landscape.html]: Is this needed because the feature is still experimental?

Comment (kul3r4) [reply src/stories/20_Features/Desktop_and_landscape.html]: not really, I noticed all the other story sample are skipping the validation. Do you know why?

File: src/stories/20_Features/Desktop_and_landscape.html (hunk 1 above)
Comment (kul3r4) [reply src/stories/20_Features/Desktop_and_landscape.html]: done

File: src/stories/20_Features/Desktop_and_landscape.html (hunk 2 above)
Comment (kul3r4) [reply src/stories/20_Features/Desktop_and_landscape.html]: done

File: src/stories/20_Features/Desktop_and_landscape.html (hunk 3 above)
Comment (kul3r4) [reply src/stories/20_Features/Desktop_and_landscape.html]: added a bigger one

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Did you see my comment: "Also: can you find (and implement) best practices for this? e.g. using srcset for images..."?

1280px is still small for fullscreen images on desktop (my screen has as resolution of 3800x2000).

**Comment by kul3r4** [comment]
@sebastianbenz I added more 2 with resolution in the srcset

**Comment by kul3r4** [comment]
@sebastianbenz updated with images of 1920

**Pull request merged**